### DISCUSSION TITLE
Exit this page — we want your research findings

### DISCUSSION THREAD

**Discussion by calvin-lau-sig7** [alphagov/govuk-design-system#2906] [Exit this page]
We've published a component to help users quickly [exit a service](https://design-system.service.gov.uk/components/exit-this-page/) and an [accompanying pattern](https://design-system.service.gov.uk/patterns/exit-a-page-quickly/). We’d welcome your feedback on how it’s being used in services.

## What we’re looking for
User research findings: tell us how the component has tested with users, what's worked and what hasn't. 

We’re particularly interested in:
- Exit this page button
- Interruption page
- Safety content

## How to share findings

You can add any findings or insights to this discussion.

We recently published [new guidance on how to share findings from users](https://design-system.service.gov.uk/community/share-research-findings/) which we hope will make it easier to format information about how the Design System is being used across services.

**Comment by owenatgov** [comment]
We have received the following feedback on the interruption page from one of the early contributors of this component:

> In regard to the discussion surrounding the creation of an interruption page for those trying to access legal aid, I would like to draw your attention to the initial reason why an exit button is required.
> 
> Those who are experiencing abuse, especially those who are living with their abuser, have limited time when trying to access support. All research from national and local support provisions, along with Domestic Homicide Reviews have concluded that the highest risk of physical harm or death occurring is during the planning stages of trying to escape the abuse and the time following the survivor/victim fleeing.
> 
> Given the critical time component, an interruption page will steal valuable seconds or even minutes from someone trying to access support and advice in the safest possible way. Having a page open which explains the risk and the reasons behind the exit this page button is sending a very clear message that this person is hiding something and a perpetrator of abuse who may see this, will take action if they feel they are losing control. By having this interruption page, the value and necessity of the exit this page button is completely undermined and could have a hugely detrimental impact on a service user.
> 
> From the Exit This Page Quickly guide, I can see that along with those who have/are experiencing domestic abuse, you have identified the following situations of risk:
> rape or sexual assault
> child abuse and neglect
> any other type of abuse
> stalking and harassment
> reporting crime or fraud (whistleblowing)
> When someone experiences these levels of trauma, the brain is altered due to the injury the trauma causes and therefore, executive functioning is impaired. This includes (among other things) working memory, the ability to process information and task initiation. All services should adopt a trauma-informed approach to ensure the correct response is given and that users can access the information needed appropriately.
> 
> An exit this page button is designed specifically with the neuroscience of trauma in mind. Someone who has or is experiencing trauma is in a constant state of survival. The logical part of the brain shuts down in order to respond to the perceived threat and mitigate harm. The exit this page button is clear and is red in colour so psychologically, the eye is drawn and the amygdala can respond to danger by clicking on it to close the page and minimise the risk.
> 
> When someone is in a state of fight or flight and the nervous system is dysregulated, they are unable to process information and any attempts to explain the need of an exit this page button will either overwhelm or put them at further risk. I pose the questions as to why it is felt this interruption page is required and who exactly is the targeted reader.
> 
> I would be happy to discuss this further to support the safety and inclusivity of a user centred approach which includes trauma informed practice.

**Comment by GWalklate** [comment]
I have seen conflicting information on the use of an exit button. In part this goes against WCAG by implementing specific localised solutions for what is essentially a browser use issue. I have yet to see an exit button that does not have the potential to put a user at risk - and they may not 'perceive' 'understand' this in the intended moment.
I was wondering how much research was done on an alternative, as focus is 'trying' to improve, get the button to work. For example, what research has been done on overlay page (pop-up to hide the real url) until user exits or accepts - here we can then educate the user, (more than you can do with a button). to open in safe browser mode, and use a keyboard shortcut/swipe to close window, risk of monitoring by abuser. This could be faster, plus reduces range of risks and can go beyond the current site.

I have seen many sites put an incognito user at risk with a button, as it loads another page in incognito rather than closes! Many home set ups will have router logs, parent controls, spyware all of which could be monitored. Many don't clear that domains history, those that do, don't clear (auto search terms record), or even the history of the google search. Many sites have documents that get downloaded (no exit button on document, plus file in downloads), Many sites signpost to other sites, some may have no exit button the user got used to, others will put it on the other side of the screen etc.

I can see why people request the button - just not sure if a 'button' is actually going to work as intended, or user to understand exactly what just happened when they clicked it (false sense of security). Other tech options? or even for government to lobby google, Microsoft for better clearer controls, or public awareness campaigns.
